scripts/windows_build_module_wheels.py

```diff
@@ -81,7 +80,7 @@ def fixup_wheel(py_envs, filepath, lib_paths:str=''):
 def fixup_wheels(py_envs, lib_paths:str=''):
     # shared library fix-up
     for wheel in glob.glob(os.path.join(ROOT_DIR, "dist", "*.whl")):
-        fixup_wheel(py_envs, wheel, ';'.join(lib_paths))
+        fixup_wheel(py_envs, wheel, lib_paths)
```

Comment:
Also reduces logic in `fixup_wheels` so that `lib_paths` is always treated as a string rather than potentially a list of strings. Valid because `fixup_wheels` is always called with a string on line 95:
```
fixup_wheels(args.py_envs, ';'.join(args.lib_paths))
```